managed/src/test/java/com/yugabyte/yw/controllers/CustomerConfigControllerTest.java

```diff
@@ -169,4 +169,65 @@ public void testDeleteInUseStorageConfig() {
     assertEquals(0, CustomerConfig.getAll(defaultCustomer.uuid).size());
     assertAuditEntry(1, defaultCustomer.uuid);
   }
+
+  @Test
+  public void testEditInUseStorageConfig() {
+    ObjectNode bodyJson = Json.newObject();
+    JsonNode data = Json.parse("{\"BACKUP_LOCATION\": \"test\", \"ACCESS_KEY\": \"A-KEY\", " +
```

Comment:
Yeah, we are using the string instead of constants lot of places. Including other tests and utils file for masking. I will create another issue to fix these separatly.

Reply:
OK, this works for me.